internal/blocktx/peer_handler.go

```diff
@@ -107,18 +109,37 @@ func (ph *PeerHandler) HandleBlockAnnouncement(msg *wire.InvVect, peer p2p.PeerI
 	}
 
 	ph.blockRequestCh <- req
-
 	return nil
 }
 
 func (ph *PeerHandler) HandleBlock(wireMsg wire.Message, _ p2p.PeerI) error {
 	msg, ok := wireMsg.(*p2p.BlockMessage)
 	if !ok {
-		ph.logger.Debug(ErrUnableToCastWireMessage.Error())
 		return ErrUnableToCastWireMessage
 	}
 
 	ph.blockProcessCh <- msg
-
 	return nil
 }
+
+func extractHeightFromCoinbaseTx(tx *sdkTx.Transaction) uint64 {
```

Comment:
This function has 2 small tests in `processor_helpers_test.go`. We can leave them there or create a separate file `peer_handler_test.go`, but just letting you know

Reply:
Moved to the new file